src/file-upload/file-upload.scss

```diff
@@ -39,7 +39,7 @@
       z-index: -1;
     }
 
-    > svg {
+    > .iui-icon {
```

Comment:
Lets rename this to something more specific

Reply:
Renamed to `.iui-file-upload-icon` in next push.